openedx/core/djangoapps/content/learning_sequences/api/tests/test_outlines.py

```diff
@@ -18,6 +18,7 @@
 import ddt
```

Comment:
How can I manually test this? I don't believe I have proctoring set up in my hosted devstack. And I'm not sure of the best way to toggle on/off exams so I can see that both still show the description correctly in the course outline.

Reply:
can you take a look at the [instructions here](https://github.com/openedx/edx-proctoring/blob/master/docs/developing.rst) to get proctoring going? Happy to hop on a call if you get stuck since there's a lot of pieces to getting this working.

Reply:
there is https://github.com/openedx/mockprock which is a fake proctoring service. I don't think anyone has set one up on hosted devstack, it would probably want to run on that same host since hosted devstack isn't going to be able to find mockproc running on your laptop
https://openedx.atlassian.net/wiki/spaces/COMM/pages/2763718660/Setup+mockprock+as+a+backend

Reply:
on the other hand you don't actually need to do much of the proctoring flow, we may have an even simpler null proctor which just says "yes go ahead" and calls nothing

Reply:
Thank you thank you!